src/components/common/PickerScreen/styles.js

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from 'react-native';
+import { whiteColor } from '../../../constants/styleConstants';
```

Comment:
Use absolute imports instead
```suggestion
import { whiteColor } from 'constants/styleConstants';
```